go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/go-bindata/go-bindata/v3 v3.1.3
 	github.com/google/cel-go v0.9.0
 	github.com/mikefarah/yq/v3 v3.0.0-20201202084205-8846255d1c37
+	github.com/operator-framework/operator-registry v1.19.5
```

Comment:
this is a circular dependency. the api project shouldn't depend on the operator-registry

Reply:
So, it shows for me that the code to do the zip ought to be shipped here and we do a follow up for opm get it from here. Tks
/hold

Reply:
HI @kevinrizza, 

That is changed. 

/hold cancel